### PR TITLE
✨ Enable installer to run on Azure Cloud Shell

### PR DESCRIPTION
**Pull request by afiune**
Updated the installer and verified it works.

Screenshot 2024-11-17 at 12 32 16 PM

Resolves https://github.com/mondoohq/installer/issues/498

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -163,8 +163,8 @@ if [ "$DISTRIBUTION" = "Darwin" ]; then
   OS="macOS"
 elif [ -f /etc/debian_version ] || [ "$DISTRIBUTION" == "Debian" ] || [ "$DISTRIBUTION" == "Ubuntu" ]; then
   OS="Debian"
-elif [ "$AWS_EXECUTION_ENV" == "CloudShell" ]; then
-  OS="AWSCloudShell"
+elif [ "$AWS_EXECUTION_ENV" == "CloudShell" ] || [ "$POWERSHELL_DISTRIBUTION_CHANNEL" == "CloudShell" ]; then
+  OS="CloudShell"
```
Comment (chris-rock) [inline install.sh:167]: How do we distinguish between the different cloud's cloudshell then?

Comment (philipbalinov) [reply install.sh:167]: LGTM in terms of fixing the issue. 

@chris-rock - do we currently have a use case where we want to differentiate between various cloud shells?
If yes, we could change the match condition to something like `[[ "$OS" = *CloudShell ]]` and prefix the different shells with the provider?

Comment (chris-rock) [reply install.sh:167]: Not right now, but in future iterations we want to make sure the script runs in AWS, Azure and GCP cloudshells.

**Comment by afiune** [comment]
Thank you both for the feedback and review, when the time comes that we need to split this into different installation per cloud environment, we should split it. For now, I didn't see the need to copy & paste the same code, so I just renamed things.

## Let's merge and release! 🚀

**Pull request merged**